components/LoginModal.tsx

```diff
@@ -50,7 +50,7 @@ export default function LoginModal({
     <Modal isOpen={isOpen} onClose={onClose ?? (() => {})} size="lg">
       <ModalOverlay />
       <ModalContent pt={5} pb={4}>
-        <form onSubmit={handleSubmit}>
+        <form onSubmit={handleSubmit} className="login-modal-form">
```

Comment:
Shouldn't need this, See https://testing-library.com/docs/queries/about#priority and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role.